hat/assets/js/apps/Iaso/domains/instances/compare/components/CompareInstanceLogs.tsx

```diff
@@ -86,15 +122,21 @@ export const CompareInstanceLogs: FunctionComponent<Props> = ({
                     logB: instanceLogsDropdown[1]?.value,
                 };
                 dispatch(
-                    redirectTo(baseUrls.compareInstanceLogs, defaultParams),
+                    redirectToReplace(
```

Comment:
This whole useEffect can be refactored i think, let's do it together when you are back